### PR TITLE
Fix "todayLabel" prop type description.

### PR DESCRIPTION
**Pull request by NiklasMM**
## Description

Fixes description and defaultValue for prop `todayLabel`

### CODE REVIEW

**Comment by dmtrKovalenko** [comment]
This is autogenerated. Comes from jsdoc comments

**Comment by NiklasMM** [comment]
aw snap... sorry. I'll fix it

**Comment by dmtrKovalenko** [comment]
1. Change description
2. run `yarn docgen`

**Comment by NiklasMM** [comment]
btw, is there a reason why the default values are in all caps in the description, but the actual default values aren't?

**Comment by dmtrKovalenko** [comment]
It is a limitation of react-docgen-typescript :(

**Comment by NiklasMM** [comment]
`yarn docgen` puts all the output into one line. I guess that should be configured in this line: https://github.com/mui-org/material-ui-pickers/blob/ff22b9e25470549534482f1c7bc7b203638f4271/docs/scripts/docgen.js#L73

Telling `stringify` to use two spaces also generates a different format. Any trick I'm missing here?

**Comment by dmtrKovalenko** [comment]
No worries. We are storing it in minfied version. 
It is github feature it is pretiffying this file

**Comment by dmtrKovalenko** [comment]
Or prettier 😟

**Comment by NiklasMM** [comment]
But the file is prettified on my file system and if I add it to the commit all lines are changed. Do I not add it to the commit? Sorry, I'm a bit confused here 😅

**Comment by dmtrKovalenko** [comment]
Just commit it :D 
No worries

**Comment by NiklasMM** [comment]
Oh, I see, there's some pre-commit magic going on. 

Thanks for walking me through it ;)

File: lib/src/wrappers/ModalWrapper.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { WrapperProps } from './Wrapper';
 import { useKeyDown } from '../_shared/hooks/useKeyDown';
 import { DialogProps as MuiDialogProps } from '@material-ui/core/Dialog';
 
-export interface ModalWrapperProps<T = {}> extends WrapperProps<T> {
+export interface ModalWrapperProps<T = {},> extends WrapperProps<T> {
```
Comment (dmtrKovalenko) [inline lib/src/wrappers/ModalWrapper.tsx]: This `,`

Comment (dmtrKovalenko) [inline lib/src/wrappers/ModalWrapper.tsx]:
```suggestion
export interface ModalWrapperProps<T = {}> extends WrapperProps<T> {
```

**Pull request merged**